src/handlebars/faq.handlebars

```diff
@@ -0,0 +1,188 @@
+{{> header title='FAQ | ' }}
```

Comment:
On staging it's a full-width page, it doesn't match the styling of other pages, missing a container?

Reply:
Good point - somehow something got lost from the end of that line when I copied it. Will fix